app/models/saved_claim.rb

```diff
@@ -115,4 +115,17 @@ def business_line
   def attachment_keys
     []
   end
+
+  def after_create_metrics
```

Comment:
@wayne-weibel Should these get moved into the monitor?

Reply:
Or no because its a different STATS_KEY?

Reply:
not in this case.  these metrics are global for all saved_claim.  since i hope more form will begin recording the 'form_start_date' seemed to make sense to put that tracking here instead of pension only (prevent other teams from needing to implement on their own)